File: angular-admin/src/app/model/config/ConfigListRequest.ts
export class ConfigListRequest {

  dataType:string;//数据类型

  keyCode:string;//key值

  means:string;//含义

  createStartDate:any = null;//配置创建的开始日期

  createEndDate:any = null;//配置创建的结束日期

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}

File: eureka-client-admin/src/main/java/com/javaweb/web/eo/config/ConfigListRequest.java
package com.javaweb.web.eo.config;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ConfigListRequest implements Serializable {
    
	private static final long serialVersionUID = 5935819846603209798L;

	private String dataType;//数据类型
    
    private String keyCode;//key值
    
    private String means;//含义
    
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条
    
}
